presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2723,6 +2731,37 @@ public void testGroupedExecution()
             assertQuery(colocatedOneGroupAtATime, chainedSharedBuildOuterJoin, expectedChainedOuterJoinResult, assertRemoteExchangesCount(2));
             assertQuery(colocatedOneGroupAtATimeDynamic, chainedSharedBuildOuterJoin, expectedChainedOuterJoinResult, assertRemoteExchangesCount(2));
 
+            //
+            // Window function
+            // ===============
+            ImmutableList.Builder<String> expectedCountValues = ImmutableList.builder();
+            expectedCountValues.addAll(nCopies(7333, "('O', 7333)"));
```

Comment:
Maybe add some filter when create the `test_grouped_window` table? Enumerate 7333 tuples in `VALUES` sounds a bit too much. Maybe make about 10% of the tuples?